src/main/java/aeneas/models/Level.java

```diff
@@ -27,25 +52,6 @@ public Level(Bullpen bullpen, boolean prebuilt) {
   public Level(Bullpen bullpen) {
     this(bullpen, true);
   }
-  
-  @Override
-  public int hashCode() {
-    return levelNumber;
-  }
-  
-  @Override
-  public boolean equals(Object o) {
```

Comment:
Because after discussion with James, I had moved the storage of metadata to be a hash map of Integer->Metadata, instead of Level->Metadata, and levels are stored in a similar map - since the Level is not a key, we don't need hashCode. 

I'm not sure if the current structure is the best, so I'm open to further ideas on that front.